src/components/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import logo from '../logo.svg';
+
+export default () => {
```

Comment:
I'm curious to see how this component displays in the React debugger. 
(https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en)

If it displays as an unnamed component, we could give it a name.